admin-notice/class-admin-notice-controller.php

```diff
@@ -41,8 +43,35 @@ public function dismiss_vip_notice() {
 		check_ajax_referer( self::DISMISS_NONCE_ACTION );
 
 		if ( isset( $_POST[ self::DISMISS_IDENTIFIER_KEY ] ) ) {
-			update_user_meta( get_current_user_id(), self::DISMISS_USER_META, sanitize_text_field( $_POST[ self::DISMISS_IDENTIFIER_KEY ] ) );
+			add_user_meta( get_current_user_id(), self::DISMISS_USER_META, sanitize_text_field( $_POST[ self::DISMISS_IDENTIFIER_KEY ] ) );
```

Comment:
Fix dismissing multiple notices bug.

Reply:
Won't this only persist the first dismissed notice, and not any more?